examples/coco/.tensorleap/dataset.py

```diff
@@ -62,35 +54,46 @@ def _download(cloud_file_path: str, local_file_path: Optional[str] = None) -> st
 def subset_images() -> List[SubsetResponse]:
     print("subset")
 
-    def load_set(coco):
+    def load_set(coco, load_union=False):
```

Comment:
```suggestion
    def load_set(coco: COCO, load_union=False):
```

Reply:
true. I'm working on the code changes that will be in the next PR that is for the README file for now

Reply:
Fixed by @daniellebenbashat